extensions/wikia/Discussions/maintenance/ForumDumper.php

```diff
@@ -111,6 +111,13 @@ public function getPages() {
 			->WHERE( 'page_namespace' )->IN( self::FORUM_NAMEPSACES )
 			->ORDER_BY( 'idx' )
 			->runLoop( $dbh, function ( &$pages, $row ) use ( &$display_order ) {
+				// A few of these properties were removed and do not appear on some wikis
+				foreach ( [ 'sticky', 'locked', 'protected' ] as $prop ) {
+					if ( !property_exists( $row, $prop ) ) {
+						$row->$prop = 0;
+					}
+				}
+
```

Comment:
AFAIK we performed a site-wide schema update in  [SUS-1690](https://wikia-inc.atlassian.net/browse/SUS-1690), so these columns should no longer appear anywhere (and they were never used)